src/Control/Lens/At.hs

```diff
@@ -469,10 +469,7 @@ instance Ord k => At (Map k a) where
   {-# INLINE at #-}
 
 instance (Eq k, Hashable k) => At (HashMap k a) where
-  at k f m = f mv <&> \r -> case r of
-    Nothing -> maybe m (const (HashMap.delete k m)) mv
-    Just v' -> HashMap.insert k v' m
-    where mv = HashMap.lookup k m
+  at k f = HashMap.alterF f k
```

Comment:
Note to self, I'll try after this is merged.

I think we can eta-reduce `f`, but I'm not sure about `k` (simplified subsumption in GHC-9.0).

Reply:
Ah, we cannot. They are flipped. Never mind.